lib/feature/penjual/views/menu_cafetaria_page.dart

```diff
@@ -396,3 +382,210 @@ class ListMenuWidget extends StatelessWidget {
     return const SizedBox.shrink();
   }
 }
+
+class BottomDaftarMenuWidget extends StatelessWidget {
+  const BottomDaftarMenuWidget({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ElevatedButton(
+      style: ElevatedButton.styleFrom(
+        minimumSize: const Size.fromHeight(44),
+        primary: const Color(0xffEA001E),
+      ),
+      child: const Text('TAMBAH MENU ATAU KATEGORI'),
+      onPressed: () async {
+        await showModalBottomSheet(
+          shape: RoundedRectangleBorder(borderRadius: BorderRadius.circular(8)),
+          context: context,
+          builder: (context) {
+            return const _ModalBottomSheet();
+          },
+        );
+        context
+            .read<MenuMakananBloc>()
+            .add(const GetMenuMakanan('0DzobjgsR7jF8qWvCoG0'));
+      },
+    );
+  }
+}
+
+// temp object for "opsi menu"
+class OpsiMenu {
+  final bool isMandatory;
+  final bool isMultipleTopping;
+  final String menuId;
+  final List<Option> option;
+  final String optionmenuId;
+  final String title;
+
+  OpsiMenu(
+    this.isMandatory,
+    this.isMultipleTopping,
+    this.menuId,
+    this.option,
+    this.optionmenuId,
+    this.title,
+  );
+}
+
+class Option {
+  final String name;
+  final int price;
+
+  Option(this.name, this.price);
+}
+// =====
+
+class OpsiMenuWidget extends StatefulWidget {
+  const OpsiMenuWidget({Key? key}) : super(key: key);
+
+  @override
+  State<StatefulWidget> createState() => _OpsiMenuWidgetState();
+}
+
+class _OpsiMenuWidgetState extends State<OpsiMenuWidget> {
+  List<OpsiMenu> menuOptions = [];
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.all(16.0),
+      width: double.infinity,
+      child: menuOptions.isEmpty
+          ? const SizedBox.shrink()
+          : SizedBox(
+              width: double.infinity,
+              height: 500,
+              child: ListView.builder(
+                shrinkWrap: true,
+                physics: const NeverScrollableScrollPhysics(),
+                itemCount: menuOptions.length,
+                itemBuilder: (context, index) {
+                  return Column(
```

Comment:
```suggestion
                  return _ItemMenuOptionWidget();
```

mungkin bisa di ubah menjadi widget sendiri biar lebih rapi. wdyt?